generators/entity-server/templates/src/main/java/package/common/field_validators.ejs

```diff
@@ -25,8 +25,12 @@ if (field.fieldValidate === true) {
     const rules = field.fieldValidateRules;
     const validators = [];
     const MAX_VALUE = 2147483647;
+    const isBlob = field.fieldType === 'byte[]';
 
-    if (rules.includes('required')) {
+    fieldTypeBlobContent
```

Comment:
Does this do anything?  I'm surprised it passed lint

Reply:
indeed this looks strange. @ruddell ejs linting is not as good as eslint. it only checks for syntax errors

Reply:
I was definitely tired when I did this. :sweat_smile: